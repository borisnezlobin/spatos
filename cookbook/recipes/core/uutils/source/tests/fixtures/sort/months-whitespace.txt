JAN
			JUNNNN
AUG 

 apr
 apr

  FEb
