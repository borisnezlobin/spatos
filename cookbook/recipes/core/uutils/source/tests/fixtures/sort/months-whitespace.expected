

JAN
  FEb
 apr
 apr
			JUNNNN
AUG 
